committee_admissions/settings/local.py

```diff
@@ -1,13 +0,0 @@
-from .development import *
```

Comment:
Deleting this file means import error for everyone that does not have it, so if the point is to delete it entirely, the `__init__.py` file in settings must be updated from using `local` to using `development`.